### PR TITLE
CountStars solution and test

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: solutions/countstars.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	res := ""
+	star := " star..."
+	if num <= 0 {
+		res = "No stars"
+	}
+	for i := 1; i <= num; i++ {
+		res = res + strconv.Itoa(i) + star
+	}
+	return res
+}
```
Comment (davidrobert99) [inline solutions/countstars.go:15]: This solution does not respect the content of the exercise.

fmt.Println(CountStars(5)) -> "1 star...2 star...3 star...4 star...5 star...".

**Review by davidrobert99 — CHANGES_REQUESTED** [review]

File: solutions/countstars.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		if i == 1 {
+			res = res + strconv.Itoa(i) + star + dots
+		} else if i != num {
+			res = res + strconv.Itoa(i) + stars + dots
+		} else {
+			res = res + strconv.Itoa(i) + stars
+		}
+	}
+	return res
+}
```
Comment (MSilva95) [inline solutions/countstars.go:27]: Please format your code

File: solutions/countstars.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	if num == 1 {
+		return strconv.Itoa(1) + star
+	}
+	for i := 1; i <= num; i++ {
+		if i == 1 {
+			res = res + strconv.Itoa(i) + star + dots
+		} else if i != num {
+			res = res + strconv.Itoa(i) + stars + dots
+		} else {
+			res = res + strconv.Itoa(i) + stars
```
Comment (MSilva95) [inline solutions/countstars.go:23]: is it possible to do it without strconv.Itoa?
If not we must specify it to allow this function!

File: tests/countstars/main.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	table := []int{1, 2, 9, -258, 0, 48, 94, 41,-56}
```
Comment (MSilva95) [inline tests/countstars/main.go:11]: add more tests here and one or two larger numbes under 1000

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: solutions/countstars.go (hunk 3 above)
Comment (zainabdnaya) [reply solutions/countstars.go:23]: Okay, I will add it in the Subject that is Allowed to use it

File: solutions/countstars.go (hunk 2 above)
Comment (zainabdnaya) [reply solutions/countstars.go]: Done

File: tests/countstars/main.go (hunk 4 above)
Comment (zainabdnaya) [reply tests/countstars/main.go]: Done

File: solutions/countstars.go (hunk 3 above)
Comment (MSilva95) [reply solutions/countstars.go:22]: You can leave the solution like it is, but it is possible to do it without the use of strconv, that was what i wanted to know.

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**